config/.metrics_rubocop.yml

```diff
@@ -0,0 +1,93 @@
+# Metrics::AbcSize
+# (Assignment Branch Condition)
+#
+# Checks that the ABC size of methods is not higher thatn the configured maximum
+
+Metrics/AbcSize:
+  Max: 15
```

Comment:
15, although the default, will probably end up being a bit too restrictive at times.